storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/Read.scala

```diff
@@ -0,0 +1,15 @@
+package com.twitter.storehaus.postgres
+
+
+/**
+  * Type class for deserializing PostgreSQL native data to scala types
+  *
+  * Created by ponkin on 10/20/16.
+  */
+trait Read[A] {
```

Comment:
can we instead follow the style of the rest of the project and use `Injection[A, PostgresValue]` here instead?